Client/core/CCommands.cpp

```diff
@@ -86,7 +86,10 @@ bool CCommands::Execute(const char* szCommand, const char* szParametersIn, bool
     // Copy szParametersIn so the contents can be changed
     char* szParameters = nullptr;
     if (szParametersIn)
+    {
+        szParameters = static_cast<char*>(alloca(strlen(szParametersIn) + 1));
```

Comment:
please use `new` instead of `alloca`:
```cpp
szParameters = new char[strlen(szParametersIn) + 1]{0};
```
If `{0}` wouldnt be supported then you can omit it.